lib/daru/io/importers/excelx.rb

```diff
@@ -0,0 +1,132 @@
+        # @param skiprows [Integer] Skips the first +:skiprows+ number of rows from the
+        #   sheet being parsed.
+        # @param skipcols [Integer] Skips the first +:skipcols+ number of columns from the
+        #   sheet being parsed.
+        # @param order [Boolean] Defaults to true. When set to true, first row of the
+        #   given sheet is used as the order of the Daru::DataFrame and data of
+        #   the Dataframe consists of the remaining rows.
+        # @param index [Boolean] Defaults to false. When set to true, first column of the
+        #   given sheet is used as the index of the Daru::DataFrame and data of
+        #   the Dataframe consists of the remaining columns.
+        #
+        #   When set to false, a default order (0 to n-1) is chosen for the DataFrame,
+        #   and the data of the DataFrame consists of all rows in the sheet.
+        #
+        # @return A +Daru::DataFrame+ imported from the given XLSX file and sheet
+        #
+        # @example Importing from a local file
+        #   path  = 'spec/fixtures/excelx/Stock-counts-sheet.xlsx'
+        #   sheet = 'Example Stock Counts'
+        #   df    = Daru::IO::Importers::XLSX.new(path, sheet: sheet).call
+        #   df
+        #
+        #   #=> <Daru::DataFrame(15x7)>
+        #              Status Stock coun  Item code        New Descriptio Stock coun Offset G/L
+        #        0          H          1        nil        nil New stock  2014-08-01        nil
+        #        1        nil          1  IND300654          2 New stock  2014-08-01      51035
+        #        2        nil          1   IND43201          5 New stock  2014-08-01      51035
+        #        3        nil          1   OUT30045          3 New stock  2014-08-01      51035
+        #       ...       ...        ...     ...           ...     ...       ...           ...
+        #
+        # @example Importing from a local file without headers
+        #   path  = 'spec/fixtures/excelx/Stock-counts-sheet.xlsx'
+        #   sheet = 'Example Stock Counts'
+        #   df    = Daru::IO::Importers::XLSX.new(path, sheet: sheet, headers: false).call
+        #   df
+        #
+        #   #=> <Daru::DataFrame(16x7)>
+        #                   0           1          2          3          4          5        6
+        #        0      Status Stock coun  Item code        New Descriptio Stock coun Offset G/L
+        #        1          H          1        nil        nil New stock  2014-08-01        nil
+        #        2        nil          1  IND300654          2 New stock  2014-08-01      51035
+        #        3        nil          1   IND43201          5 New stock  2014-08-01      51035
+        #        4        nil          1   OUT30045          3 New stock  2014-08-01      51035
+        #       ...       ...        ...     ...           ...     ...       ...           ...
+        #
+        # @example Importing from a remote URL
```

Comment:
@v0dro I believe that `daru-io` should think about it as a complementary yet separate gem. 
It is a necessary compromise we've already discussed with Athitya: whether we document exact modules that do the work (`Daru::IO::Whatever`) or their generated methods? (`DataFrame.from_whatever`)
My answer is the former. 
Let's think about somebody who uses daru-io as a foundation to add another format, say, FITS (it is astronomy data).
What they do?
1. `require 'daru-io'`
2. Create `Daru::IO::FITS` class by a simple straightforward instructions, and document it.
3. Call `Daru::IO.register_io_module`.

Of course, we **can** ask them to implement pseudo-docs for `from_fits`, YARD allows this... But do we really faithfully **want** to?..

So, I believe `Daru::IO` docs should be saying something like... "Blah, blah, when you include this and that, you receive a ton of `Daru::DataFrame.from_this_and_that`, please refer to `Daru::IO::ThisAndThatImporter` for docs".

It is one option. Another one is stop polluting DF with `from_foo`/`to_bar` methods, and define for `Daru::IO` API like `Daru::IO::FromExcel.new('path').call` / `Daru::IO::ToExcel.new(df).call('path')`.